llmfoundry/data/text_data.py

```diff
@@ -151,6 +151,8 @@ def __init__(self,
             sampling_method=sampling_method,
             sampling_granularity=sampling_granularity,
             batching_method=batching_method,
+            allow_unsafe_types=False,
+            replication=None,
```

Comment:
I'd error on the side of passing them in as args at line 111 then setting the default there.

Reply:
Done. PTAL.